### DISCUSSION TITLE
Choosing only one bounding box

### DISCUSSION THREAD

**Discussion by AfroCheremsha** [mikel-brostrom/yolo_tracking#642] [Q&A]
Hi! In track.py from 194 to 219 lines you draw bboxes per frame.

I want to use OpenCV Mouse Click Event to choose only one bbox that will be drawn. In other words, I want to track the selected object. I don't need BBoxes around the other objects. How can i did it inside track.py?

Thank you for your attention.

**Comment by mikel-brostrom** [comment]
Sure. Save the (x,y) pixel values of you click an compare them to the bboxes in each track. If you click is inside you can dismiss the rest of the tracks
